.ci.yaml

```diff
@@ -2057,8 +2057,17 @@ targets:
         ["devicelab"]
     scheduler: luci
 
-  - name: Mac_ios native_ui_tests_ios32
-    builder: Mac_ios native_ui_tests_ios32
+  - name: Mac_ios32 native_ui_tests_ios32
```

Comment:
I also added this builder as it existed in the luci config but not here

Reply:
I am not sure what you mean here. I see `Mac_ios native_ui_tests_ios32` already exists in the yaml.

Reply:
I fixed this. My LSC just checked for the task name in the infra config, and over-wrote them. This overwrote some of the Mac_android builders and made them Linux_android. I reran the script, and now the diff is the expected diff.